app/services/conversation/telemetry.py

```
@@ -0,0 +1,88 @@
+
+@dataclass
+class TelemetryRecord:
+    user_id: str
+    interface: str
+    intent: str
+    request: str
+    confidence: float
+    resolution: Dict[str, Any]
+    state_summary: Dict[str, Any]
+    outcome: str
+    created_at: datetime = datetime.utcnow()
```

Ruff (0.13.2)

Do not perform function call `datetime.utcnow` in dataclass defaults

(RUF009)